### ISSUE TITLE
Storage 'store: gcs' specified, but GCP access is disabled

### ISSUE THREAD

**Issue by MaoZiming** [skypilot-org/skypilot#2134]
After I merge recent commits on master branch, I got
```
sky.exceptions.ResourcesUnavailableError: Storage 'store: gcs' specified, but GCP access is disabled. To fix, enable GCP by running `sky check`
```
on the spot controller. When I ran `sky check`, I got
```
GCP: enabled
```
I redid the GCP setup with no error. It is working on the branch before merging.

**Comment by concretevitamin** [comment]
cc @romilbhardwaj @landscapepainter

**Comment by MaoZiming** [comment]
This seems to be fixed after #2135

**Comment by Michaelvll** [comment]
Ahh, yes. It was a bug introduced by #2116 and should be fixed by #2135. Closing now. Thanks for reporting this @MaoZiming!
